lib/core/src/client/preview/client_api.js

```diff
@@ -11,14 +11,7 @@ const defaultDecorateStory = (getStory, decorators) =>
   );
 
 export default class ClientApi {
-  constructor({
-    channel,
-    storyStore = new StoryStore(),
-    decorateStory = defaultDecorateStory,
-  } = {}) {
-    // channel can be null when running in node
-    // always check whether channel is available
-    this._channel = channel;
```

Comment:
channel here is only used by RN?

Reply:
This code (`client_api`) wasn't previously used by RN.

`_channel` is always there in existing code that uses the client API, but not used by the client API, so given the way I was going to initialize this code in the RN context (without passing the channel as it doesn't exist yet), I figured we get rid of if from this code, so no-one would try and use it in the future.